federation/registry/cluster/strategy_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/kubernetes/federation/apis/federation"
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/testapi"
```

Comment:
@sttts do you prefer importing `"k8s.io/kubernetes/pkg/api/testing"` in eatch `_test.go` or just importing it in `pkg/api/testing`? I can see arguments both sides.